Source/FlowEditor/Public/DetailCustomizations/FlowOwnerFunctionRefCustomization.h

```diff
@@ -48,6 +48,4 @@ class FFlowOwnerFunctionRefCustomization : public IFlowCuratedNamePropertyCustom
 	static TArray<FName> GetFlowOwnerFunctionRefs(const UFlowNode_CallOwnerFunction& FlowNodeOwner, const UClass& ExpectedOwnerClass);
 
 	static bool IsFunctionUsable(const UFunction& Function, const UFlowNode_CallOwnerFunction& FlowNodeOwner);
-	static bool DoesFunctionHaveExpectedParamType(const UFunction& Function, const UFlowNode_CallOwnerFunction& FlowNodeOwner);
```

Comment:
the fixed model for CallOwnerFunction configuration does not need this function any longer.